Building DAG of jobs...
Using shell: /bin/bash
Provided cores: 4
Rules claiming more threads will be scaled down.
Job stats:
job                  count
-----------------  -------
all                      1
run_python_script        1
total                    2

Select jobs to execute...

[Thu Sep 19 16:55:15 2024]
rule run_python_script:
    input: /home/administrator/phd/PhagePleats_out/foldseek_out/search_result.tsv, data/presence_absence_c_40_tm_30_4100.csv, data/4100_phage_ICTV_metadata.csv, data/models, /home/administrator/phd/PhagePleats_out
    output: /home/administrator/phd/PhagePleats_out/taxa_predictions.csv
    jobid: 4
    reason: Missing output files: /home/administrator/phd/PhagePleats_out/taxa_predictions.csv
    resources: tmpdir=/tmp

[Thu Sep 19 16:55:31 2024]
Error in rule run_python_script:
    jobid: 4
    input: /home/administrator/phd/PhagePleats_out/foldseek_out/search_result.tsv, data/presence_absence_c_40_tm_30_4100.csv, data/4100_phage_ICTV_metadata.csv, data/models, /home/administrator/phd/PhagePleats_out
    output: /home/administrator/phd/PhagePleats_out/taxa_predictions.csv

RuleException:
CalledProcessError in file /home/administrator/phd/PhagePleats/Snakefile, line 97:
Command 'mkdir -p /home/administrator/phd/PhagePleats_out &&  /home/administrator/programs/miniforge3/envs/CaudoAuto/bin/python3.10 /home/administrator/phd/PhagePleats/.snakemake/scripts/tmpabcyx_i5.phagepleats.py' returned non-zero exit status 1.
  File "/home/administrator/phd/PhagePleats/Snakefile", line 97, in __rule_run_python_script
  File "/home/administrator/programs/miniforge3/envs/CaudoAuto/lib/python3.10/concurrent/futures/thread.py", line 58, in run
Shutting down, this might take some time.
Exiting because a job execution failed. Look above for error message
Complete log: .snakemake/log/2024-09-19T165515.639009.snakemake.log
